source/mainnet/net/desktop-wallet/install-ledger-app.rst

```diff
@@ -109,7 +109,11 @@ You now have to install a custom certificate on the Ledger to ensure that it tru
 
 #. Extract the files from the ZIP folder to a folder on your computer.
 
-#. Press the right button while you reconnect the Ledger to the computer, and hold it down until the Ledger says **recovery**.
+#. Now you need to start recovery mode on the Ledger device.
+
+   - On the **Nano S**, press the *right* button and hold it down while you reconnect the Ledger to the computer until the Ledger says **recovery**.
+
+   - On the **Nano S Plus**, press the *left* button and hold it down while you reconnect the Ledger to the computer. Press both buttons to select **recovery**.
```

Comment:
```suggestion
   - On the **Nano S Plus**, press the *left* button and hold it down while you reconnect the Ledger to the computer. Navigate to **Recovery mode** and press both buttons to enter **recovery mode**.
```
The same for the other entries. The difference is that you have to do some navigation on the S Plus.